From fe89f9bc4055fd6055ce8792a715a40a6c75ba44 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Thu, 17 Dec 2020 13:54:04 +0100
Subject: [PATCH] tools: Set mode for new file /tmp/mt76-test-%s

Set the file system mode for the newly created file /tmp/mt76-test-%s,
this is mandatory according to the man page and fixes a compile error
with glibc.

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 tools/eeprom.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/eeprom.c
+++ b/tools/eeprom.c
@@ -77,7 +77,7 @@ mt76_eeprom_create_file(void)
 		return -1;
 	}
 
-	fd = open(eeprom_file, O_RDWR | O_CREAT | O_EXCL);
+	fd = open(eeprom_file, O_RDWR | O_CREAT | O_EXCL, 00644);
 	if (fd < 0)
 		goto out;
 
